OpenProblemLibrary/ASU-topics/setCalculus/stef/stef2_5p4.pg

```diff
@@ -28,34 +29,32 @@ DOCUMENT();        # This should be the first executable line in the problem.
 
 loadMacros(
   "PGstandard.pl",
-  "PGchoicemacros.pl",
-  "extraAnswerEvaluators.pl",
-  "MathObjects.pl",
+  "PGML.pl",
   "PGcourse.pl"
 );
 
-$b = random(1,8,1);
+Context("Interval");
+$b = random(1,8);
+$answer= Interval("[$b,Inf)");
+$f= Formula("sqrt(x-$b)");
 
-TEXT(beginproblem());
 
-$showPartialCorrectAnswers = 1;
+BEGIN_PGML
 
-TEXT(EV3(<<'EOT'));
-Let
-\[ f(x) = \sqrt{x - $b}.  \]
-Use \{helpLink("interval notation")\} to indicate where \( f \) is continuous.
+Let [`f(x)`] =  [`[$f]`].   Use [@ helpLink("interval notation") @]*  to indicate where [`f(x)`] is continuous.
 
-$BR$BR
+Domain of continuity:[___________]{$answer}{15}
```

Comment:
For this, you don't need both underscores `_` and the 15 at the end.  I would recommend just putting in one `_`.